README.md

```diff
@@ -22,6 +22,14 @@ Einstein on `OC1.Docker` can share a file with the `federalists` group and the r
 Copy `federatedgroups` folder into apps folder of your owncloud. 
 Change **sharing.managerFactory** config entry to **OCA\\FederatedGroups\\ShareProviderFactory** inside *config.php* file.
 
+```
+  'sharing.managerFactory' => 'OCA\\FederatedGroups\\ShareProviderFactory',
+  'sharing.remoteShareesSearch' => 'OCA\\OpenCloudMesh\\ShareeSearchPlugin',
```

Comment:
This is already mentioned in the OCM repo:

https://github.com/pondersource/oc-opencloudmesh/blob/main/README.md?plain=1#L22

I think we should not repeat it since people may get confused.